src/util/storybook-docs/StoryLayout.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import FDS from 'lib/dictionary/js/styleConstants';
 
-export const StoryWrapper = ({ children, className, ...rest }) => (
+export const StoryWrapper = ({ children, className = '', ...rest }) => (
```

Comment:
important to note: importing this non Typescript file into a Typescript file caused an error to be throw, because I wasn't passing a className. It's interesting that Typescript interprets a destructure as *required* props automatically.